opencog/attention/WAImportanceDiffusionAgent.cc

```diff
@@ -25,18 +25,21 @@
  */
 
 #include <opencog/util/Config.h>
+#include <opencog/atomspace/AtomSpace.h>
 #include <opencog/attention/atom_types.h>
 #include <opencog/attentionbank/AttentionBank.h>
+#include <opencog/attentionbank/StochasticImportanceDiffusion.h>
 
 #include "WAImportanceDiffusionAgent.h"
 
 using namespace opencog;
 
 
 WAImportanceDiffusionAgent::WAImportanceDiffusionAgent(CogServer& cs) :
-    ImportanceDiffusionBase(cs)
+    ImportanceDiffusionBase(cs), _as(opencog::server_atomspace())
```

Comment:
well, at this point, it doesn't much matter, but this really should be cs.getAtomSpace() so that the cogserver and the agent both use the same atomspace.